### PR TITLE
Increase docker proxy configuration logs

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

To provide further information when we configure the proxy settings for docker 

#### Which issue(s) this PR fixes:

Motivates by # [sc-76762]

#### Does this PR introduce a user-facing change?

```release-note
Adds further log information for Docker Proxy settings configuration
```

### CODE REVIEW

File: scripts/common/docker.sh [hunk 1]
```diff
@@ -160,12 +160,17 @@ information.\n\nhttps://help.replicated.com/docs/kb/developer-resources/devicema
 }
 
 docker_configure_proxy() {
+    log "Configuring docker proxy"
     local previous_proxy=$(docker info 2>/dev/null | grep -i 'Http Proxy:' | awk '{ print $NF }')
     local previous_no_proxy=$(docker info 2>/dev/null | grep -i 'No Proxy:' | awk '{ print $NF }')
+    log "Previous proxy: ($previous_proxy)"
+    log "Previous no proxy: (previous_no_proxy)"
```
Comment (emosbaugh) [inline scripts/common/docker.sh:167]:
```suggestion
    log "Previous no proxy: ($previous_no_proxy)"
```

File: scripts/common/docker.sh [hunk 2]
```diff
@@ -160,12 +160,17 @@ information.\n\nhttps://help.replicated.com/docs/kb/developer-resources/devicema
...
     if [ "$PROXY_ADDRESS" = "$previous_proxy" ] && [ "$NO_PROXY_ADDRESSES" = "$previous_no_proxy" ]; then
+        log "No changes were found. Proxy configuration still the same"
         return
     fi
 
+    log "Updating proxy configuration with new changes"
```
Comment (emosbaugh) [inline scripts/common/docker.sh:173]:
```suggestion
    log "Updating proxy configuration: HTTP_PROXY=${PROXY_ADDRESS} NO_PROXY=${NO_PROXY_ADDRESSES}"
```

File: scripts/common/docker.sh [hunk 3]
```diff
@@ -178,6 +183,8 @@ docker_configure_proxy() {
         echo "Environment=\"HTTP_PROXY=${PROXY_ADDRESS}\" \"NO_PROXY=${NO_PROXY_ADDRESSES}\"" >> $file
     fi
 
+    log "Environment=\"HTTP_PROXY=${PROXY_ADDRESS}\" \"NO_PROXY=${NO_PROXY_ADDRESSES}\""
+
```
Comment (emosbaugh) [inline scripts/common/docker.sh:187]:
```suggestion
```

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: scripts/common/docker.sh (hunk 3 above)
Comment (camilamacedo86) [reply scripts/common/docker.sh:187]: We have not this info in the logs today 
See that we just add to the file.
Would be not great we are able to check in the logs for the support cases?

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**